literature.bib

```diff
@@ -372,7 +367,29 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{FK:POS24,
+	title       = {{Life Span of SAT Techniques}},
+	author      = {Mathias Fleury and Daniela Kaufmann},
+	booktitle   = POS,
+	institution = arXiv,
+	doi         = {10.48550/arXiv.2402.01202},
+	year        = 2024,
+	ek-tags     = {satisfiability, preprocessing}
+}
 
+@article{YOL+:FSE24,
```

Comment:
YOLG instead of YOL+? Or are we not doing this anymore when having 4 authors?

Reply:
Yes, I started implementing this. The difficult part is the conference. Currently, there are many false positives, when the check is enabled.